### PR TITLE
[Add Package] Lyrebird

### PR DESCRIPTION
**Pull request by D-Brox**
This PR adds `lyrebird`
Fixed dependencies and requests from previous PR

### CODE REVIEW

**Comment by D-Brox** [comment]
The codacy analysis is saying it needs 2 spaces for the  list-item indend, but in the README there are already 2 spaces

**Comment by wizard-28** [comment]
> The codacy analysis is saying it needs 2 spaces for the list-item indend, but in the README there are already 2 spaces

Oh, Ignore it then.

File: packages/lyrebird/lyrebird.pacscript [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name="lyrebird"
+pkgname="lyrebird"
```
Comment (wizard-28) [inline packages/lyrebird/lyrebird.pacscript:2]:
```suggestion

```
Not needed as there are no extra suffixes like `-git` to this pacscript. And remember to refactor all instances of `pkgname` -> `name`

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]
Works!

**Pull request merged**